playbooks/adhoc/upgrades/upgrade.yml

```diff
@@ -281,7 +281,7 @@
     - name: Upgrade node packages
       command: yum update -y {{ openshift.common.service_type }}-node{{ openshift_version }}
     - name: Restart node services
-      service: name="{{ openshift.common.service_type }}-node" state=restarted
+      service: name="{{ openshift.common.service_type }}-node" enabled=yes state=restarted
```

Comment:
OK I will add a separate step late in the process to ensure everything is started/enabled.